src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/relationships-property/cardinality-selector/cardinality-selector.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, Input, EventEmitter, Output, OnInit, SimpleChanges, OnChanges } from '@angular/core';
+import { DotRelationshipCardinality } from '@portlets/content-types/fields/shared/dot-relationship-cardinality.model';
+import { DotMessageService } from '@services/dot-messages-service';
+import { RelationshipService } from '@portlets/content-types/fields/content-type-fields-properties-form/field-properties/relationships-property/services/relationship.service';
+
+@Component({
+    providers: [],
+    selector: 'dot-cardinality-selector',
+    templateUrl: './cardinality-selector.component.html'
+})
+export class CardinalitySelectorComponent implements OnInit, OnChanges {
+    @Input()
+    cardinalityIndex: number;
```

Comment:
if the component already have "cardinality" in the number, why we just don't call this "index", is not like we're calling "disabled" "cardinalityDisabled" :D